tmux_help.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+echo """
+TMUX basic commands: see https://github.com/gpakosz/.tmux
+- commands require a <prefix>, then a key: Ctrl+a = <prefix>
+- a 'window' is like a tab
+- a 'pane' is a subdivision of current terminal, like tiling
+You can use keybindings below or click with mouse to select active pane/window, resize etc
+
+Windows (ie tabs)
+  <prefix> C-h and <prefix> C-l let you navigate windows
+  <prefix> Tab brings you to the last active window
```

Comment:
Could we also add instructions for how to create new windows? Seems like something we might want to do often.